CS-476: Added readme with steps to create Docker container for React.JS applications

raghavsharma007 opened the pull request.

raghavsharma007 committed:
CS-476: Added Dockerizing react app readme file

raghavsharma007 committed:
CS-476: Added dockerizing react app - Corrected file structure

parasteh commented:
Thank you @raghavsharma007  for the contribution. Please update the readme. I'd like you to please send a link to your document. here is a guide on how to create the link:
[](https://github.blog/2013-01-31-relative-links-in-markup-files/)

For better structure, you can add it under DevOps. there is a good tutorial on the Reactjs you can add that as well. 

Great job!

raghavsharma007 committed:
CS-476: Added dockerizing react app - Added reactJS documentation

raghavsharma007 commented:
Hi @parasteh , It is already added under DevOps folder. Also, this topic(Docker + Reactjs) is different from the document(Kubernetes), so I will create another pull request for that document. 
For now, I have updated the pull request to added steps to create and start reactJS application and added the documentation as well.
Thanks

parasteh commented:
Great, Thanks @raghavsharma007
